docs/explanations/motd_messages.md

```diff
@@ -2,21 +2,47 @@
 
 When the Ubuntu Pro Client (`pro`) is installed on the system, it delivers
 custom messages on ["Message of the Day" (MOTD)](https://wiki.debian.org/motd).
-Those messages are generated directly by two different sources.
+Those messages are generated directly by three different sources.
 
-## Python-scripted MOTD
+## MOTD notifying about available updates
 
 The [update-notifier](https://wiki.ubuntu.com/UpdateNotifier) delivers a script
-called `apt_check.py`. With regards to Ubuntu Pro, this script is responsible
-for:
+via the `update-notifier-common` package called
+`/usr/lib/update-notifier/apt_check.py.
+With regards to Ubuntu Pro, this script is responsible for:
   
 * Informing the user about the status of one of the ESM services; `esm-apps` if
   the machine is an LTS series, or `esm-infra` if the series is in ESM mode.
 * Showing the number of `esm-infra` or `esm-apps` packages that can be upgraded
   on the machine.
 
-For example, here is the output of the `apt_check.py` script on a LTS machine
-when both of those services are enabled:
+`update-notifier` always was responsible to add information about potential
+updates to MOTD to raise user awareness, with the existence of pro they are
+just more differentiated.
+
+Note that if you run `apt_check.py` directly it might give you rather
+unreadable output as it is meant for program use, you can add `--human-readable`
+to get the same info you see in MOTD.
+
+### Machine is unattached
+
+On a machine that runs a Ubuntu release for which the `esm-apps` service
+is available, but not yet attached to Ubuntu Pro in any way there will
+only be a hint that there could be more available through ESM.
```

Comment:
```suggestion
be a message notifying the user that there may be more security updates available through ESM.
```

Reply:
Although "hint" isn't wrong, it may have some connotations that we don't intend to imply. I think message would generally be clearer and have less room for interpretation.